typescript/infra/scripts/warp-routes/export-warp-configs.ts

```diff
@@ -18,6 +20,12 @@ async function main() {
   const warpIdsToCheck = warpRouteId
     ? [warpRouteId]
     : Object.keys(warpConfigGetterMap);
+  const eslint = new ESLint({
+    fix: true,
+    overrideConfigFile:
+      '/Users/leyu/Desktop/Code/hyperlane/hyperlane-registry/eslint.config.js',
```

Comment:
todo: figure out how to import this from the registry, or just hardcode the relevant rules